src/Elasticsearch.Net/Domain/RequestParameters/RequestParameters.Generated.cs

```diff
@@ -1613,30 +1613,6 @@ public class AnalyzeRequestParameters : FluentRequestParameters<AnalyzeRequestPa
 	{
 		public override HttpMethod DefaultHttpMethod => HttpMethod.POST;
 		
-		///<summary>The name of the analyzer to use</summary>
-		public AnalyzeRequestParameters Analyzer(string analyzer) => this.AddQueryString("analyzer", analyzer);
-		
-		
```

Comment:
Thanks for taking a look @gmarz. I've changed the generated `RequestParameters` to be partial classes so that I can patch these methods back in. Thoughts?